pr_line_related_po_line/i18n/es.po

```diff
@@ -0,0 +1,44 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* pr_line_related_po_line
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 16.0+e\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-03-03 18:34+0000\n"
+"PO-Revision-Date: 2023-03-03 18:34+0000\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: pr_line_related_po_line
+#: model:ir.model,name:pr_line_related_po_line.model_purchase_order_line
+msgid "Purchase Order Line"
+msgstr "Línea de pedido de compra"
+
+#. module: pr_line_related_po_line
+#: model:ir.model.fields,field_description:pr_line_related_po_line.field_purchase_requisition_line__purchase_order_line_ids
+msgid "Purchase Order Lines"
+msgstr "Líneas de pedido de compra"
+
+#. module: pr_line_related_po_line
+#: model:ir.model,name:pr_line_related_po_line.model_purchase_requisition_line
+#: model:ir.model.fields,field_description:pr_line_related_po_line.field_purchase_order_line__purchase_requisition_line_id
+msgid "Purchase Requisition Line"
+msgstr "Línea solicitud de compra"
+
+#. module: pr_line_related_po_line
+#: model:ir.model.fields,field_description:pr_line_related_po_line.field_purchase_order_line__display_name
+#: model:ir.model.fields,field_description:pr_line_related_po_line.field_purchase_requisition_line__display_name
+msgid "Display Name"
```

Comment:
Are you sure this is the correct position?

Terms are sorted alphabetically when re-exporting the file.

Is this the order the file was generated?

Reply:
Neither replied nor applied

Reply:
fixed   [here](https://github.com/Vauxoo/addons-vauxoo/pull/1594/files#diff-50a8e001fe03eefa20cc8023cc12b96f2f6494756e567db67dfdd8e4a7a969b5)